config/routes.rb

```diff
@@ -15,6 +15,8 @@
   get '/dashboard/items', to: "items#index"
   put '/dashboard/item', to: "items#update"
   delete '/dashboard/item', to: "items#destroy"
+  get '/dashboard/items/new', to: "items#new"
+  post '/dashboard/items/new', to: "items#create"
```

Comment:
See above comment - I don’t think we want the /new on this

Reply:
Resolved per above.